task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -186,6 +195,13 @@ spec:
         echo "WARNING: provided deprecated BUILDER_IMAGE parameter has no effect."
       fi
 
+      ca_bundle=/mnt/trusted-ca/ca-bundle.crt
+      if [ -f "$ca_bundle" ]; then
+        echo "INFO: Using mounted CA bundle: $ca_bundle"
+        cp -vf $ca_bundle /etc/pki/ca-trust/source/anchors
```

Comment:
Perhaps we can symlink instead of copy to avoid creating another copy of the entire bundle in RAM.

For now we end up with 3 copies at least:
1. The config map
2. The file in `/etc/pki/ca-trust/source/anchors`
3. The resulting bundle file after running `update-ca-trust`

Maybe we can even mount the config map directly into `/etc/pki/ca-trust/source/anchors` ?

Reply:
maybe, but we now that this approach works and I don't think the memory overhead is too big.
I think we should proceed with this PR for unblocking @ashwindasr

Reply:
How hare is it to test using `ln -s` instead of `cp` ?